app/helpers/feature_flag.rb

```diff
@@ -1,7 +1,9 @@
 module FeatureFlag
-  extend self # rubocop:disable Style/ModuleFunction
+  class << self
+    delegate :enabled?, :exist?, to: Flipper
 
-  def enabled?(feature_name, *args)
-    Flipper[feature_name].enabled?(*args)
+    def accessible?(feature_flag_name, *args)
+      feature_flag_name.blank? || !exist?(feature_flag_name) || enabled?(feature_flag_name, *args)
```

Comment:
Right now we call `accessible?` in `StoriesController`, where we pass in `@page.feature_flag_name`. In `Page` we define this method to return `"page_#{slug}"`. So am I missing something or can this never be `blank?`?

Reply:
You're right: the way I'm calling it now it won't be blank, but I want this helper class to be resilient for any kind of use case: I don't want to always assume feature flag name will be passed in!